### PR TITLE
Separate overview pages for sync and async

### PR DESCRIPTION
**Pull request by AlexJerabek**
Also includes links to the Office Add-ins ref doc space.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[8d5eae1](https://github.com/OfficeDev/office-scripts-docs-reference/commits/8d5eae1e8517e9afb05fc53a3a9c0d5e86797d7e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docs-ref-autogen/excel-async/overview.md](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Overview/docs/docs-ref-autogen/excel-async/overview.md) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/overview.md](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Overview/docs/docs-ref-autogen/excel/overview.md) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/overview/overview.md](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Overview/docs/docs-ref-autogen/overview/overview.md) | :white_check_mark:Succeeded | |
[generate-docs/scripts/postprocessor.ts](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Overview/generate-docs/scripts/postprocessor.ts) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C17%5Cb7e1f975-26d5-fb7f-b635-d5c5bc9f9be0%5CPullRequest%5C202006172052000093-42%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=JnRq2Yaj4KCXDLCCwCDztFEBU80PkDZCUXaVCZdVKOE%3D&st=2020-06-17T20%3A50%3A20Z&se=2020-07-18T20%3A55%3A20Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/b7e1f975-26d5-fb7f-b635-d5c5bc9f9be0?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/docs-ref-autogen/excel-async/overview.md [hunk 1]
```diff
@@ -1,28 +1,32 @@
...
 ---
 
-# Office Scripts API reference
+# Office Scripts Async API reference
 
-The Office Scripts API lets you automate common tasks in Excel on the web. Use this reference documentation to learn more about the classes, methods, and other types available for your scripts. All the objects accessible through Office Scripts can be found in the table of contents on the left of the page.
+The Office Scripts Async API supports older scripts made during the Office Scripts preview phase. Use this reference documentation to learn more about the classes, methods, and other types used by these older scripts. All the objects accessible through Office Scripts can be found in the table of contents on the left of the page.
+
+> [!IMPORTANT]
+> We strongly recommend creating new scripts with the standard Office Scripts APIs. If you're making or editing a new script, please switch to the [non-async version](?view=office-scripts-async) of the APIs.
```
Comment (ElizabethSamuel-MSFT) [inline docs/docs-ref-autogen/excel-async/overview.md:12]: What page are you linking to? Also, is the view correct?

File: docs/docs-ref-autogen/excel-async/overview.md [hunk 2]
```diff
@@ -1,28 +1,32 @@
...
 - [Ranges](/javascript/api/office-scripts/excel/excel.range) are used to create and place [Tables](/javascript/api/office-scripts/excel/excel.table), [Charts](/javascript/api/office-scripts/excel/excel.chart), [Shapes](/javascript/api/office-scripts/excel/excel.shape), and other data visualization or organization objects.
-- A [Worksheet](/javascript/api/office-scripts/excel/excel.worksheet) contains arrays filled with those objects that are present in the individual sheet.
-- A [Workbook](/javascript/api/office-scripts/excel/excel.workbook) contains arrays of some of those data objects for the entire Workbook.
+- A [Worksheet](/javascript/api/office-scripts/excel/excel.worksheet) contains collections of those data objects (such as a [ChartCollection](/javascript/api/office-scripts/excel/excel.chartcollection)) that are present in the individual sheet.
+- [Workbooks](/javascript/api/office-scripts/excel/excel.workbook). contain collections of some of those data objects (such as a [TableCollection](/javascript/api/office-scripts/excel/excel.tablecollection)) for the entire [Workbook](/javascript/api/office-scripts/excel/excel.workbook).
```
Comment (ElizabethSamuel-MSFT) [inline docs/docs-ref-autogen/excel-async/overview.md:23]:
```suggestion
- [Workbooks](/javascript/api/office-scripts/excel/excel.workbook) contain collections of some of those data objects (such as a [TableCollection](/javascript/api/office-scripts/excel/excel.tablecollection)) for the entire [Workbook](/javascript/api/office-scripts/excel/excel.workbook).
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[f8702d1](https://github.com/OfficeDev/office-scripts-docs-reference/commits/f8702d12824c702748aef2c64597e291bdb415b2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docs-ref-autogen/excel-async/overview.md](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Overview/docs/docs-ref-autogen/excel-async/overview.md) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/overview.md](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Overview/docs/docs-ref-autogen/excel/overview.md) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/overview/overview.md](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Overview/docs/docs-ref-autogen/overview/overview.md) | :white_check_mark:Succeeded | |
[generate-docs/scripts/postprocessor.ts](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Overview/generate-docs/scripts/postprocessor.ts) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C17%5Cb7e1f975-26d5-fb7f-b635-d5c5bc9f9be0%5CPullRequest%5C202006172116047176-42%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=uHzCBLqHgXg4MoFs%2Bujs6hjlLvgjTCrM8g5N93ZyHAU%3D&st=2020-06-17T21%3A13%3A57Z&se=2020-07-18T21%3A18%3A57Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/b7e1f975-26d5-fb7f-b635-d5c5bc9f9be0?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/docs-ref-autogen/excel-async/overview.md (hunk 1 above)
Comment (AlexJerabek) [reply docs/docs-ref-autogen/excel-async/overview.md]: Yeah, wrong view... This is linking to the current page, but with a different moniker.

**Comment by AlexJerabek** [comment]
Thanks for the feedback @ElizabethSamuel-MSFT. Everything should be addressed.

**Pull request merged**